src/current/_includes/v23.2/cdc/mux-rangefeed.md

```diff
@@ -0,0 +1,28 @@
+`MuxRangefeed` is a subsystem that improves the performance of rangefeeds with scale. It significantly reduces the overhead of running {% if page.name == "create-and-configure-changefeeds.md" %} [rangefeeds](#enable-rangefeeds) {% else %} [rangefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds). {% endif %} Without `MuxRangefeed` enabled the number of RPC streams is proportional with the number of ranges in a table. For example, a large table could have tens of thousands of ranges. With `MuxRangefeed` enabled, this proportion improves so that the number of RPC streams is relative to the number of nodes in a cluster.
+
+We recommend enabling its functionality with the `changefeed.mux_rangefeed.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}), which will be enabled by default in future major releases.
```

Comment:
Is this necessary: ", which will be enabled by default in future major releases"?

The only mentions in the docs of future changes is when something is deprecated and will be removed in the future.

Reply:
I think this is necessary to let users know that turning this on is worthwhile because it will be a default feature in the furture.